commands/filescommands/objectcommands/uploaddir.go

```diff
@@ -0,0 +1,235 @@
+package objectcommands
```

Comment:
Well, we are creating/uploading objects, and not containers, so it felt more natural to be based on object. I'm not sure the decision is 100% final.